tasks/skylab/PairedTagUtils.wdl

```diff
@@ -187,6 +187,7 @@ task ParseBarcodes {
         File atac_fragment
         Int nthreads = 1
         String cpuPlatform = "Intel Cascade Lake"
+        String docker_path
```

Comment:
This didn't need to be made backwards compatible since it's only used in PairedTag.wdl currently.